README.md

```diff
@@ -728,6 +728,41 @@ _You can enable the following settings in Xcode by running [this script](resourc
   let dictionary: [String: Any] = [:]
   ```
 
+* <a id='single-line-expression-braces'></a>(<a href='#single-line-expression-braces'>link</a>) The opening brace following a single-line expression should not wrap to a newline.[![SwiftFormat: braces](https://img.shields.io/badge/SwiftFormat-braces-7B0051.svg)](https://github.com/nicklockwood/SwiftFormat/blob/master/Rules.md#braces)
```

Comment:
Do you have any thoughts on what you would propose for multi-line expressions? I think starting here is great. I'm wondering what direction we may go to understand if it may inform this rule.

Reply:
Style for multi-line expressions is somewhat covered by other existing rules (and we have the SwiftFormat `wrapMultilineStatementBraces` rule enabled)
- "Put the open curly brace on the next line so the first executable line doesn't look like it's another parameter.": https://github.com/airbnb/swift#long-function-declaration
- Not explicitly mentioned in the prose, but included in examples: https://github.com/airbnb/swift#multi-line-conditions

It couldn't hurt to explicitly mention the multiline case in this rule, if that seems like it would be more clear.

Reply:
> It couldn't hurt to explicitly mention the multiline case in this rule, if that seems like it would be more clear.

I like the idea of being explicit about braces for all types of expressions in this rule, though it seems like SwiftFormat will only autocorrect single-line expressions?

Reply:
I added a prose rule for the multi-line case in d2860b9